### ISSUE TITLE
A11y Enhancement - Links Should Have Discernible Text

### ISSUE THREAD

**Issue by aitchiss** [DDDEastMidlandsLimited/dddem-web#99]
### Describe the feature

Raising this issue in relation to Issue #50 . 
Axe report show several instances of icons (e.g. Twitter, LinkedIn) being used without any discernible text for screen reader users - see example screenshot below. This is particularly noticeable on the Home Page footer and the Speakers page. Proposed fix is to add relevant aria-labels to these icons where they appear.

I am happy to take this on and raise a PR for it.

<img width="1251" alt="Screenshot 2019-10-07 at 20 06 47" src="https://user-images.githubusercontent.com/20773163/66341113-7e6fa180-e93e-11e9-871b-848774135675.png">

### Advantages

* Links currently attached to icons, e.g. Speaker Twitter accounts and LinkedIn profiles will become accessible to screen reader users

#### Acceptance Criteria

- [ ] Axe or similar a11y testing tool should show no issues with icon links
- [ ] When tabbing through links with screen reader active, appropriate content should be announced on icon links e.g. "{ Speaker name: } Twitter"

**Comment by HERBOD7** [comment]
@wordshaker and @aitchiss I can do it.

**Comment by aitchiss** [comment]
@HERBOD7 @wordshaker I think this was closed by #100 ?

**Comment by wordshaker** [comment]
@HERBOD7 - @aitchiss is correct. Sorry I'll close teh issue
